x-pack/plugins/observability/public/components/app/observability_status/observability_status.stories.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiProvider } from '@elastic/eui';
+import { ObservabilityStatusBoxes, ObservabilityStatusProps } from './observability_status_boxes';
+
+export default {
+  title: 'app/ObservabilityStatusBoxes',
+  component: ObservabilityStatusBoxes,
+  decorators: [
+    (Story: ComponentType) => (
+      <EuiProvider>
```

Comment:
This is only a temporal solution until https://github.com/elastic/kibana/pull/121901 is merged